src/courtroom.cpp

```diff
@@ -4572,7 +4571,7 @@ void Courtroom::on_sfx_edit_requested()
 {
   QString p_path = ao_app->get_real_path(ao_app->get_character_path(current_char, "soundlist.ini"));
   if (!file_exists(p_path)) {
-    p_path = ao_app->get_base_path() + "soundlist.ini";
+    p_path = ao_app->get_real_path(VPath("soundlist.ini"));
```

Comment:
This won't work as you would expect it to because if the vpath isn't found, the path is not resolvable and will be returned as blank. So you need to get the path of the parent directory.

Reply:
I don't get it, can you elaborate or make a suggestion?

Reply:
It works because get_real_path resolves it through all mounted paths and finds the first one

Reply:
If there is no soundlist.ini in any mounted paths, then there will not be a path resolved, and editing the global soundlist.ini will fail. Here is my suggestion:

```suggestion
    p_path = ao_app->get_real_path(VPath("soundlist.ini"));
  }
  if (!file_exists(p_path)) {
    p_path = ao_app->get_base_path() + "soundlist.ini";
  }
```